src/main/java/core/selenium/WebElementAction.java

```diff
@@ -71,14 +84,14 @@ public void selectOnDropdownMenu(final WebElement webElement, final String value
      *
      * @param webElement the text box
      * @param value the value to choose
+     * @param displayedElementPath the element to select
      */
-    public void selectOnAutoCompleteTextBox(final WebElement webElement, final String value) {
+    public void selectOnAutoCompleteTextBox(final WebElement webElement, final String value,
+                                            final String displayedElementPath) {
         driver = getWebDriverManager().getDriver();
         wait = getWebDriverManager().getWebDriverWait();
```

Comment:
The driver wait could be on the base test so you only call that instance instead of the drivermanager.

Reply:
changed it